content/guides/collaborations/connect-slack-to-group-collabs/5-connect-box-functions.md

```diff
@@ -0,0 +1,413 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Connect Bot to Box
+
+We're now handling and processing events coming from Slack, then obtaining all
+information needed to connect with Box users and groups. We now need to
+connect that functionality to Box functions.
+
+In this step we'll expand several functions from the last step to incorporate
+new Box features.
+
+* Instantiate a Box client.
+* Add a Box user to a Box group.
+* Remove a Box user from a Box group.
+* Fetch a Box group ID from a group name.
+* Add content that is shared to a group.
+
+## Instantiate a Box Client
+
+To make calls to the Box APIs, you'll first need to set up a Box client.
+
+<Choice option='programming.platform' value='node' color='none'>
+
+Within `process.js`, replace the `// INSTANTIATE BOX CLIENT` comment at the top
+with the following.
+
+```javascript
+const configJSON = JSON.parse(fs.readFileSync(path.resolve(__dirname, './config.json')));
+const sdk = boxSDK.getPreconfiguredInstance(configJSON);
+const client = sdk.getAppAuthClient('enterprise');
```

Comment:
This can be written a lit simpler:

```js
const boxConfig = require('./boxConfig.json');
const client = box.getPreconfiguredInstance(boxConfig)
  .getAppAuthClient('enterprise');
``` 

(I also think that if we have a `boxConfig.json` it would clearly differentiate it from the Slack configuration.